doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,414 @@
+
+.. code-block:: text
+
+    subsection void fraction
+      set mode                         = qcm
+      set qcm sphere equal cell volume = false
+      set qcm sphere diameter          = 0.005326
+      set read dem                     = true
+      set dem file name                = dem
+      set l2 smoothing length          = 0.02663
+      set quadrature rule              = gauss-lobatto
+      set n quadrature points          = 5
+    end
+
+
+CFD-DEM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We enable grad-div stabilization in order to improve local mass conservation. If we were using PCM and SPM void fraction schemes, the void fraction time derivative should be disabled as the time variation of the void fraction will lead to unstable simulations. The source of such instability is the first term of the continuity equation :math:`\rho_f \frac{\partial \varepsilon_f}{\partial t}`, which is stiff and unstable for the slightest temporal discontinuity of the void fraction and as :math:`\Delta t \to 0`. However, as we are using the QCM void fraction scheme, this term can be enabled. Usually, this term is neglected, however; disabling this term affects the results as we are no longer solving for the actual `Volume Averaged Navier-Stokes equations <../../../theory/multiphase/cfd_dem/unresolved_cfd-dem.html>`_. Therefore, we should not neglect this term based on numerical reasoning without any physical explanation.
+
+.. code-block:: text
+
+    subsection cfd-dem
+      set grad div                      = true
+      set void fraction time derivative = true
+      set drag force                    = true
+      set buoyancy force                = true
+      set shear force                   = true
+      set pressure force                = true
+      set drag model                    = rong
+      set coupling frequency            = 100
+      set grad-div length scale         = 0.005
+      set vans model                    = modelA
+    end
+
+
+We determine the drag model to be used for the calculation of particle-fluid forces. We enable buoyancy, drag, shear and pressure forces. For drag, we use the Rong model to determine the momentum transfer exchange coefficient. The VANS model we are solving is model A. Other possible option is model B.
+
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection non-linear solver
+      subsection fluid dynamics
+        set solver           = inexact_newton
+        set tolerance        = 1e-8
+        set max iterations   = 10
+        set verbosity        = verbose
+        set matrix tolerance = 0.75
+      end
+    end
+
+We use the ``inexact_newton`` solver as to avoid the reconstruction of the system matrix at each Newton iteration. For more information about the non-linear solver, please refer to the `Non Linear Solver Section <../../../parameters/cfd/non-linear_solver_control.html>`_
```

Comment:
It does not matter to use the inexact newton here I would use the regular one and not worry about it tbh.